### ISSUE TITLE
Alpha clamping

### ISSUE THREAD

**Issue by RobinCamarasa** [j-min/Dropouts#3]
Good afternoon,
I have been working on your implementation of Variational Dropout and something seems weird to me in the forward pass
```python
            # Clip alpha
            self.log_alpha.data = torch.clamp(self.log_alpha.data, max=self.max_alpha)
            alpha = self.log_alpha.exp()
```
The code above seems to clamp the log of alpha and not the alpha itself which is not what we expect. Do I misinterpret you code or the theory here ?
